code/obj/item/cloths.dm

```diff
@@ -190,32 +190,35 @@ ABSTRACT_TYPE(/obj/item/cloth/handkerchief)
 	user.put_in_hand_or_drop(the_bandana)
 	boutput(user, "<span class='notice'>You tie \the [src] together to make \a [the_bandana].</span>")
 
-/obj/item/cloth/handkerchief/white
+ABSTRACT_TYPE(/obj/item/cloth/handkerchief/colored)
```

Comment:
Sorry it's taken me so dang long to start reviewing this. Slowly making my way through.

Why are handkerchiefs changed in this PR? The changes are fine but I don't see how they're related to storage?

Reply:
:p There was a bug with handkerchiefs bought from spacebux purchases, where the spacebux handkerchief would be added to storage when created, but it's really a random spawner and so it would delete itself in New(), adding something disposed to storage and runtiming. This was the fix for it, didn't want to do a QDELETED() check in storage since that shouldn't be happening anyways, but can change